src/util/psbt/map/global.rs

```diff
@@ -61,13 +61,13 @@ impl Map for PartiallySignedTransaction {
         }
 
         // Serializing version only for non-default value; otherwise test vectors fail
-        if self.version > 0 {
+        if self.version as u32 > 0 {
```

Comment:
I didn't realize we could use `as` like this. Perhaps we should add 
```rust

    fn to_standard(self) -> u32 {
        self as u32
    }
``` 

(feels like I'm stealing @Kixunil's review tips, I must be learning :)

Reply:
implemented as `to_raw(self)`